src/daemon/main.c

```diff
@@ -1093,7 +1093,23 @@ static int get_hostname(char *buf, size_t buf_size) {
     return gethostname(buf, buf_size);
 }
 
-static void get_netdata_configured_variables() {
+static int get_valid_update_every(int update_every) {
```

Comment:
Considering the name that function has, should we write an `info` when the input is different of expected values/?

Reply:
I saw you are raising an alert outside the function.